src/app/component/user/components/profile/calendar/calendar.component.ts

```diff
@@ -32,92 +37,113 @@ export class CalendarComponent implements OnInit, OnDestroy {
     isCurrentDayActive: false
   };
 
-  constructor(private translate: TranslateService) {}
+  constructor(private translate: TranslateService,
+              private languageService: LanguageService) {}
 
   ngOnInit() {
     this.bindDefaultTranslate();
     this.subscribeToLangChange();
     this.buildCalendar();
   }
 
-  private getDaysInMonth(iMonth, iYear): number {
+  public getDaysInMonth(iMonth, iYear): number {
     return new Date(iYear, iMonth + 1, 0).getDate();
   }
 
-  private subscribeToLangChange(): void {
+  public subscribeToLangChange(): void {
     this.langChangeSub = this.translate.onDefaultLangChange.subscribe((res) => {
       const translations = res.translations.profile.calendar;
       this.daysNameLong = translations.days;
       this.months = translations.months;
       this.monthAndYearName = this.months[this.currentMonth] + ' ' + this.currentYear;
```

Comment:
use es6 concatenation - here and anywhere!

Reply:
Fixed